src/net/sourceforge/kolmafia/persistence/ItemFinder.java

```diff
@@ -277,7 +325,9 @@ private static void filterNameList(List<String> nameList, Match filterType) {
           itemId != -1
               && !ItemDatabase.getAttribute(
                   itemId, EnumSet.of(Attribute.TRADEABLE, Attribute.MATCHABLE, Attribute.QUEST))
-              && !NPCStoreDatabase.contains(itemId));
+              && !NPCStoreDatabase.contains(itemId)
```

Comment:
`itemAvailable` checks this only if the user has `autoSatisfyWithNPCs`, which I think is what is probably desired here, so this line can be removed

Reply:
This is a break in behaviour as `autoSatisfyWithNPCs` is false by default, but I think it makes the way `ItemFinder` works less confusing (and also maybe we should adjust the `autoSatisfy` defaults).

Reply:
As in my below comment I think we should leave this to be conservative